app/vmstorage/main.go

```diff
@@ -222,6 +222,16 @@ func RequestHandler(w http.ResponseWriter, r *http.Request) bool {
 		}()
 		return true
 	}
+	if path == "/internal/force_flush" {
+		authKey := r.FormValue("authKey")
+		if authKey != *forceMergeAuthKey {
```

Comment:
Could you add `forceFlushAuthKey` instead of using `forceMergeAuthKey`?

Reply:
Done